README.md

```diff
@@ -24,21 +24,21 @@ Below you will find everything you need to complete the certification process an
 
 ## <a name='Repository'></a>Repository Structure
 * `README.md`             -This is the main documentation file which should contain everything you need to complete the certification process. If anything is unclear, please refer to this document first.
-* `gum-gum-htb.js` - This is your partner module file, by default it contains a template divided into multiple sections which need to be completed.
-* `gum-gum-htb-validator.js` - This is the validator file for the configuration object that will be passed into your module.
-* `gum-gum-htb-exports.js` - A file that contains all of the modules exports (i.e. any functions that need to be exposed to the outside world).
+* `gumgum-htb.js` - This is your partner module file, by default it contains a template divided into multiple sections which need to be completed.
```

Comment:
Please change `gumgum` back to `gum-gum`, as it is a naming convention we follow.